setup.py

```diff
@@ -123,6 +123,7 @@
             'freertos = pyocd.rtos.freertos:FreeRTOSPlugin',
             'zephyr = pyocd.rtos.zephyr:ZephyrPlugin',
```

Comment:
Would you mind moving `zephyr` to the end of this list? I should have kept it in sorted order… Thanks.

Reply:
Of course! In fact, in my first version I did just that, then thought of not introducing spurious changes.